streamx-common/src/main/scala/com/streamxhub/streamx/common/enums/ExecutionMode.java

```diff
@@ -115,4 +115,12 @@ public static boolean isKubernetesMode(Integer value) {
         return Lists.newArrayList(KUBERNETES_NATIVE_SESSION.getMode(), KUBERNETES_NATIVE_APPLICATION.getMode());
     }
 
+    public static boolean isRemoteMode(Integer value) {
```

Comment:
I have a small question, is it necessary to add the isRemoteMode method? 😅

The isYarnMode, isKubernetesMode methods exist because yarn mode and k8s  mode each contain multiple specific modes, such as yarn mode contains yarn_per_job, yarn_application, yarn_session.
But standalone, as I understand it, has only one sub-mode, so I suggest removing this set of methods.